app/controllers/pre-payment.controller.test.js

```diff
@@ -254,31 +254,7 @@ describe('Pre payment controller', () => {
 
           controller(req, res)
 
-          sinon.assert.calledWith(mockResponse.response, req, res, 'start/start', { continueUrl: `/pay/${productExternalId}/confirm` })
-        })
-      })
-    })
-    describe('The new payment link journey is disabled', () => {
-      describe('The product has reference enabled', () => {
-        it('Should call the payment links V1 reference controller', () => {
```

Comment:
Any reason these are removed? We still have V1 payment links journey

Reply:
👍